test/services/actionService.test.js

```diff
@@ -0,0 +1,18 @@
+
+const actionsService = require('../../server/services/actionsService')
+describe('actionsService', () => {
+  test('actions service returns a actions object containing an array', async () => {
+    expect.assertions(1)
+    const actions = await actionsService.get()
+    expect(Array.isArray(actions)).toEqual(true)
+  })
+
+  test('actions service object contains an array where the first element has an id of 1', async () => {
```

Comment:
Again, the test description doesn't match the assertion.

Reply:
Fixed